src/services/encryptPassword.js

```diff
@@ -0,0 +1,10 @@
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+
+module.exports.encryptPassword = async function (password) {
+    var pass;
```

Comment:
Please use let instead of var
```suggestion
    let pass;
```